protobuf-provider/src/main/java/io/confluent/kafka/schemaregistry/protobuf/ProtobufSchema.java

```diff
@@ -1293,6 +1373,20 @@ private static MessageDefinition toDynamicMessage(
         }
       }
     }
+    for (ExtensionsElement extension : messageElem.getExtensions()) {
+      for (Object elem : extension.getValues()) {
+        if (elem instanceof Integer) {
+          int tag = (Integer) elem;
+          message.addExtensionRange(tag, tag + 1);
+        } else if (elem instanceof IntRange) {
+          IntRange range = (IntRange) elem;
+          message.addExtensionRange(range.getStart(), range.getEndInclusive() + 1);
+        } else {
```

Comment:
Will this overflow if `tag == Integer.MAX_VALUE` for instance ? Similar for `range.getEndInclusive`.

Reply:
Good question, the maximum tag number allowed by Protobuf is 2^29 - 1, or 536,870,911, which is less than Integer.MAX_VALUE